src/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -193,23 +224,21 @@ See the LICENSE file in the project root for more information.
   <Target Name="CreateLib"
     Inputs="@(LibInputs)"
     Outputs="$(SharedLibrary)"
-    DependsOnTargets="$(CreateLibDependsOn)">
+    DependsOnTargets="$(CreateLibDependsOn);$(IlcDynamicBuildPropertyDependencies)"
+    >
 
     <ItemGroup>
       <CustomLibArg Include="/out:$(SharedLibrary)" Condition="'$(OS)' == 'Windows_NT'" />
       <CustomLibArg Include="-crs $(SharedLibrary)" Condition="'$(OS)' != 'Windows_NT'" />
       <CustomLibArg Include="@(LibInputs->'%(Identity)')" />
     </ItemGroup>
-
     <MakeDir Directories="$(NativeIntermediateOutputPath)" />
     <WriteLinesToFile File="$(NativeIntermediateOutputPath)lib.rsp" Lines="@(CustomLibArg)" Overwrite="true" Condition="'$(OS)' == 'Windows_NT'" />
-
     <MakeDir Directories="$([System.IO.Path]::GetDirectoryName($(SharedLibrary)))" />
-
     <Exec Command="$(CppLibCreator) @&quot;$(NativeIntermediateOutputPath)lib.rsp&quot;" Condition="'$(OS)' == 'Windows_NT'" />
     <Exec Command="$(CppLibCreator) @(CustomLibArg, ' ')" Condition="'$(OS)' != 'Windows_NT'" />
   </Target>
 
-  <Import Project="$(MSBuildThisFileDirectory)\Microsoft.NETCore.Native.Publish.targets" Condition="'$(NativeCompilationDuringPublish)' != 'false'" />
+  <Import Project="$(IlcBuildIntegrationTargetsDir)Microsoft.NETCore.Native.Publish.targets" Condition="'$(NativeCompilationDuringPublish)' != 'false'" />
```

Comment:
Do we still need IlcBuildIntegrationTargetsDir ? It seems like that it can be just MSBuildThisFileDirectory everewhere.

Reply:
(Or not needed in some places.)